README.md

```diff
@@ -237,7 +237,6 @@ We're going to use rbenv to install Ruby for us. But first, there is more housek
 ```bash
 sudo apt-get install libssl-dev
 sudo apt-get install libsqlite3-dev
-sudo apt-get install build-essential
```

Comment:
Oh they added this back in as a default dependency with a random software update for 16.04, and if they install 17.04 it's also there. It wasn't included originally in 16.04! Problem solved!